src/js/jsx/search/SearchBar.jsx

```diff
@@ -91,7 +93,8 @@ define(function (require, exports, module) {
             this.setState({
```

Comment:
The call to `_update` gets the options for the search module. I added it in here because Recent Documents options weren't being added if I just got all the options when registering the module. It sounds like making it a `StoreWatchMixin` would take care of that so I'll try it out.